src/qualimap/qualimap_rnaseq/config.vsh.yaml

```
@@ -0,0 +1,117 @@
+name: "qualimap"
+keywords: [RNA-seq, quality control, QC Report]
+description: |
+  Qualimap RNA-seq QC reports quality control metrics and bias estimations 
+  which are specific for whole transcriptome sequencing, including reads genomic 
+  origin, junction analysis, transcript coverage and 5’-3’ bias computation.
+links:
+  homepage: http://qualimap.conesalab.org/
+  documentation: http://qualimap.conesalab.org/doc_html/analysis.html#rna-seq-qc
+  issue_tracker: https://bitbucket.org/kokonech/qualimap/issues?status=new&status=open
+  repository: https://bitbucket.org/kokonech/qualimap/commits/branch/master
+references:
+  doi: https://10.1093/bioinformatics/btv566
+license: GNU GPL v2
```

see https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository

```suggestion
license: GPL-2.0
```